net-print/npadmin/npadmin-0.8.7-r2.ebuild

```diff
@@ -18,8 +19,14 @@ SLOT="0"
 PATCHES=(
 	"${FILESDIR}"/${P}-stdlib.patch
 	"${FILESDIR}"/${P}-gcc6.patch
+	"${FILESDIR}"/${P}-makefile.patch
 )
 
+src_prepare() {
+	default
+	eautoconf
```

Comment:
The commit message says `eautoreconf`, but you are using `eautoconf`, is this correct?

Reply:
Yes. Miswrote in commit. But as eautoreconf doesn't harm in this place, switched content of commit to match the patch.